app/src/main/java/com/infomaniak/mail/ui/main/menu/MoveFragment.kt

```diff
@@ -41,6 +51,10 @@ class MoveFragment : MenuFoldersFragment() {
 
     private val createFolderDialog by lazy { initNewFolderDialog() }
 
+    private lateinit var searchResultsAdpater: FolderAdapter
```

Comment:
Avoid `lateinit` as much as possible, it can be a source of bugs, here you have an observe for the filter results, if it goes first it crashes.

Reply:
done